### PR TITLE
[PHP] Run unit tests on 'tests' directory

### PR DESCRIPTION
**Pull request by ackintosh**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

By #7200, the `test` directory is completely cleaned and all the test code is integrated into `tests`.  So we have to run unit tests which in the `tests` directory.

### CODE REVIEW

**Comment by ackintosh** [comment]
https://travis-ci.org/swagger-api/swagger-codegen/builds/342288933#L5767

🤔 💦 

> [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.2.1:exec (bundle-test) on project PhpPetstoreClientTests: Command execution failed. Cannot run program "vendor/bin/phpunit tests" (in directory "/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/php/SwaggerClient-php"): error=2, No such file or directory -> [Help 1]
> [ERROR] 
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR] 
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
> [ERROR] 
> [ERROR] After correcting the problems, you can resume the build with the command
> [ERROR]   mvn <goals> -rf :PhpPetstoreClientTests

**Comment by wing328** [comment]
@ackintosh no worry. I'll take a look later this week.

File: samples/client/petstore/php/SwaggerClient-php/pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <executable>vendor/bin/phpunit</executable>
+                            <executable>vendor/bin/phpunit tests</executable>
```
Comment (wing328) [inline samples/client/petstore/php/SwaggerClient-php/pom.xml]: You need to do something like the following instead:
```
                        <configuration>
                            <executable>vendor/bin/phpunit</executable>
                            <arguments>
                                <argument>tests</argument>
                            </arguments>
                        </configuration>
```

Comment (ackintosh) [reply samples/client/petstore/php/SwaggerClient-php/pom.xml]: Thanks 😭 ✨

**Comment by ackintosh** [comment]
https://travis-ci.org/swagger-api/swagger-codegen/builds/343299617?utm_source=github_status&utm_medium=notification
> 1) Swagger\Client\PetApiTest::testInvalidArgument
> Failed asserting that exception of type "\InvalidArgumentException" is thrown.

https://github.com/swagger-api/swagger-codegen/blob/47614bb76555cc09b7b70a8250538c15d9812827/samples/client/petstore/php/SwaggerClient-php/tests/PetApiTest.php#L381-L385

hmm...
I think this means that we have different issue: `PetApi::findPetsByStatus()` accepts an empty array 😕 
`PetApi::findPetsByStatus()` should throw an exception if an empty array is passed.

**Pull request merged**